### PR TITLE
Remove temp code-color fix, and general code style cleanup

### LINKED ISSUE #1929: Change default <code> color from RED to something else
In general, the color of `<code>` elements under the new site design is the default `pretty-print` red. Red should be used parsimoniously.

> Screen Shot 2019-09-20 at 09 12 52

How about matching the flutter.dev color (##008f83, which happens to be the same color used on angulardart.dev and, I believe, was the same color that we used to have as part of the previous site design), or one that is just a bit darker in order to have [sufficient contrast](https://webaim.org/resources/contrastchecker) (#006555):

> image

### PR DESCRIPTION
**Pull request by chalin**
- Closes #1929: introduce proper code-color fix; a temporary fix was introduced in #2000
- Does a general `code` and `pre` style cleanup

No significant style changes are introduced except for `nocode` code blocks (`pre`) now use the default font color rather than the `code` font color. This is more consistent with the prettifier that doesn't color code text that isn't recognized as belonging to a particular lexical class.

Staged at https://dart-dev-staging-1.firebaseapp.com, e.g., see
- https://dart-dev-staging-1.firebaseapp.com/guides/language/language-tour#symbols vs
  https://dart.dev/guides/language/language-tour#symbols
 (The "Runes" section just above "Symbols" has an embedded DartPad.)
- https://dart-dev-staging-1.firebaseapp.com/web/debugging#walkthrough vs
  https://dart.dev/web/debugging#walkthrough

cc @johnpryan @kwalrath

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "site-shared"]
   path = site-shared
   url = https://github.com/dart-lang/site-shared.git
-  branch = master
+  branch = chalin-scss-bs-config-1018
```
Comment (chalin) [inline .gitmodules]: Using a branch so as to avoid forcing changes on other sites that are using master.

File: src/_assets/css/_variables.scss [hunk 2]
```diff
@@ -115,47 +115,3 @@ $alert-info-bg:           #e7f8ff;
...
-//== Dropdown
-
-$dropdown-divider-bg: #bbbbbb;
-$dropdown-link-active-color: transparent;
-$dropdown-link-active-bg: transparent;
-
-//== Popovers
-
-$popover-font-size: $font-size-base;
-$popover-border-color: rgba(0,0,0,0);
```
Comment (chalin) [inline src/_assets/css/_variables.scss]: Moved into (the site-local) `_bootstrap_config.scss`.

File: src/_assets/css/main.scss [hunk 3]
```diff
@@ -159,63 +158,6 @@ input, select {
...
-    font-family: inherit;
-    font-weight: inherit;
-  }
-}
-code {
-  background-color: transparent;
-  color: $code-color;
-  font-size: 100%;
-  padding: 0;
-}
```
Comment (chalin) [inline src/_assets/css/main.scss]: Moved into (the site-local) `_bootstrap_config.scss`.

**Comment by kwalrath** [comment]
@legalcodes can you review this, or should we ask @johnpryan?

**Comment by chalin** [comment]
@johnpryan: ETA for a review?  (No pressure, it's just so I can plan my activities.)

**Comment by johnpryan** [comment]
I'll review this by Friday, thanks for your patience

File: src/_assets/css/_bootstrap_config.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+//== Breadcrumbs
+
+$breadcrumb-divider: 'chevron_right';
+$breadcrumb-item-padding: .25rem;
+
+//== Card
+
+$card-border-width: 0;
+// $card-border-color: rgba(0,0,0,0);
```
Comment (johnpryan) [inline src/_assets/css/_bootstrap_config.scss]: can you remove the commented out code in this file?

File: src/_assets/css/_code.scss [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  font-size: 0.75rem;
+  font-weight: $font-weight-normal;
+  padding: 30px; // TODO(chalin): 1.5rem;
+}
+
+@import 'code_shared';
+
+// code {
+//   background-color: transparent;
+// }
```
Comment (johnpryan) [inline src/_assets/css/_code.scss]: same here, can you remove the commented out code?

**Comment by chalin** [comment]
Thanks, I'll delete that commented out code in a followup PR, at the same time that we switch back to using site-shared at master.

**Pull request merged**